extensions/common/vault/vault-aws/src/test/java/org/eclipse/edc/vault/aws/AwsSecretsManagerVaultTest.java

```diff
@@ -63,16 +66,31 @@ void storeSecret_shouldSanitizeKey() {
 
         vault.storeSecret(KEY, value);
 
-        verify(secretClient).createSecret(CreateSecretRequest.builder().name(SANITIZED_KEY)
+        verify(secretClient).updateSecret(UpdateSecretRequest.builder().secretId(SANITIZED_KEY)
                 .secretString(value).build());
     }
 
     @Test
-    void storeSecret_shouldNotOverwriteSecrets() {
-        var value = "value";
+    void storeSecret_shouldUpdateSecretIfExist() {
+        String value = "value";
+
+        vault.storeSecret(KEY, value);
+
+        verify(secretClient).updateSecret(UpdateSecretRequest.builder().secretId(SANITIZED_KEY)
+                .secretString(value).build());
+        verify(secretClient, never()).createSecret(any(CreateSecretRequest.class));
```

Comment:
why not use `verifyNoMoreInteractions`?

Reply:
Not in particular. I'm going to use `verifyNoMoreInteractions`